packages/frontend/src/js/app/components/map/map.js

```diff
@@ -292,7 +298,9 @@ export default {
             /**
              *
              */
-            layersControl: null
+            layersControl: null,
+
+            layerToDisplay: this.layerName
```

Comment:
J'ai pas l'impression qu'il soit nécessaire de rajouter layerToDisplay dans data() {...} vu qu'on a pas l'air de pouvoir modifier son état

Tu peux utiliser directement `this.mapLayers[this.layerName]` en dessous